clarity_ext/context.py

```diff
@@ -196,6 +196,104 @@ def commit(self):
             obj.commit()
 
 
+class FakingEntityMonkey:
+    """
+    Prepare genologics.entities for an entity based mock by
+    overwriting methods to use fake data instead.
+    NOTE! This monkey is not complete, right now it's just adapted for the
+    dilution script. It has to be updated for future extended testing.
+    Process.input_output_maps is not yet overwritten.
+    """
+
+    def __init__(self):
+
+        self._overwrite_entity()
+        self._overwrite_string_descriptor()
+        self._overwrite_process()
+        self._overwrite_lims()
+        self._overwrite_entity_list_descriptor()
+
+    def reset(self):
+        EntityListDescriptor.__get__ = self.entity_list_descriptor_get
```

Comment:
The purpose of this class is to be able to crate an entity based mock. This is not the mock itself